documentation/api/notation.rst

```diff
@@ -237,10 +237,13 @@ With the flexibility context, we aim to describe the system in which the flexibl
 - ``inflexible-device-sensors`` ― power sensors that are relevant, but not flexible, such as a sensor recording rooftop solar power connected behind the main meter, whose production falls under the same contract as the flexible device(s) being scheduled
 - ``consumption-price-sensor`` ― the sensor which defines costs/revenues of consuming energy
 - ``production-price-sensor`` ― the sensor which defines cost/revenues of producing energy
-- ``site-power-capacity`` (defaults to the Asset attribute ``capacity_in_mw``)
+- ``site-power-capacity`` ― defaults to the Asset attribute ``capacity_in_mw`` ― maximum/minimum achievable power at the grid connection point.
+- ``site-consumption-capacity`` ― defaults to ``site-power-capacity`` or otherwise to the Asset attribute ``consumption_capacity_in_mw`` ― maximum consumption power at the grid connection point.
+- ``site-production-capacity`` ― defaults to ``site-power-capacity`` or otherwise to the Asset attribute ``production_capacity_in_mw`` ― maximum production power at the grid connection point.
```

Comment:
This needs an update. We can mention that a default can be set as an asset attribute, and we can mention how we take the minimum. Then we can add a footnote (see lines 219 and 222 for an example how to do that) with:
- an example of how these parameters could be used (i.e. a connection capacity in MVA and asymmetric contracted transport capacities in kW).
- what happens if a parameter is not set (also not as a default), e.g. what happens if the site-production-capacity isn't set.